README.md

```diff
@@ -40,14 +40,15 @@ Also see Apple's [Local and Remote Notification Programming Guide][notification]
 
 This library requires [Go 1.6](https://golang.org/dl/) or better.
```

Comment:
Good point. Will need to do some testing. Also would like to improve the error message #28.